src/ontology/edits/oeo-social.omn

```diff
@@ -199,6 +199,8 @@ Class: OEO_00000045
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A producer is an agent that makes goods.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/314
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/370",
         rdfs:label "producer"
```

Comment:
did we agree on naming it "producer" or "energy producer"?

Reply:
this is the global producer as a subclass of agents (that just got a def update). Not just for energy

Reply:
aha okay...and you did put organizational energy producer as a subclass of organization, right?

Reply:
yes

Reply:
okay thanks I think I got it now ;-)